idd/Energy+.idd.in

```diff
@@ -49491,87 +49491,6 @@ Coil:Cooling:Water:DetailedGeometry,
        \note higher than the coil design outlet air temperature, then the design inlet water temperature 
        \note value is reset to coil design outlet air temperature minus 5.0 DeltaC.
 
-CoilSystem:Cooling:Water,
```

Comment:
Not sure what happened when this was merged in. This is somewhat nit-picking, but this object should stay where it is in the idd.